contracts/modules/Checkpoint/ERC20DividendCheckpointFactory.sol

```diff
@@ -24,6 +26,7 @@ contract ERC20DividendCheckpointFactory is ModuleFactory {
         description = "Create ERC20 dividends for token holders at a specific checkpoint";
         compatibleSTVersionRange["lowerBound"] = VersionUtils.pack(uint8(0), uint8(0), uint8(0));
         compatibleSTVersionRange["upperBound"] = VersionUtils.pack(uint8(0), uint8(0), uint8(0));
+        logicContract = _logicContract;
```

Comment:
shouldn't we add 0x0 check for logicContract ?

Reply:
It is checked in the proxy contract.

Reply:
I was preferring the check here because if the logic contract is `0x0` here then factory deployment is just a wastage of the gas because there is no way to change the `logicContract` in the  factory contract